blaze/server/server.py

```diff
@@ -279,6 +279,11 @@ def compserver(dataset):
 
     try:
         result = compute(expr, {leaf: dataset})
+
+        if iscollection(expr.dshape):
+            result = into(list, result)
```

Comment:
minor nit: for new code we're using `odo`, which is the same as `into` except the argument order is reversed, like this:

```python
into(list, result)  # old code
odo(result, list)  # like cp result list
```